packages/shared/components/popups/StakingConfirmation.svelte

```diff
@@ -29,15 +34,23 @@
     let showTooltip = false
     let tooltipAnchor: unknown
 
-    const activeAirdrops =
+    const activeAirdropsParticipatedIn =
         $selectedAccountParticipationOverview?.participations
             .map((p) => getAirdropFromEventId(p.eventId))
             // Falsy values (undefined, null, ...) are filtered out from the array
             .filter(Boolean) || []
 
+    const activeAirdrops = STAKING_EVENT_IDS.filter((x) => x).map((x) => getAirdropFromEventId(x))
```

Comment:
What is our plan for `STAKING_EVENT_IDS` b/c it currently contains both the Shimmer and Assembly event IDs? This statement only makes sense if we manually include/exclude the IDs in that array based on if the airdrop has a new and active period.

Also why the `.filter((x) => x)` code?

Reply:
This makes sense, although just looks weird upon first glance. I suppose it's needed though..